machinelearning/h2o4gpu/README.md

```diff
@@ -0,0 +1,89 @@
+# h2o4gpu
+
+H2O4GPU is an open source, GPU-accelerated machine learning package with APIs in Python and R that allows anyone to take advantage of GPUs to build advanced machine learning models. A variety of popular algorithms are available including Gradient Boosting Machines (GBM’s), Generalized Linear Models (GLM’s), and K-Means Clustering.
+
+https://www.h2o.ai/products/h2o4gpu/
+
+H2O4GPU is a collection of GPU solvers by H2Oai with APIs in Python and R. The Python API builds upon the easy-to-use scikit-learn API and its well-tested CPU-based algorithms. It can be used as a drop-in replacement for scikit-learn (i.e. import h2o4gpu as sklearn) with support for GPUs on selected (and ever-growing) algorithms. H2O4GPU inherits all the existing scikit-learn algorithms and falls back to CPU algorithms when the GPU algorithm does not support an important existing scikit-learn class option. The R package is a wrapper around the H2O4GPU Python package, and the interface follows standard R conventions for modeling.
+
+
+The h2o4gpu code is at: https://github.com/h2oai/h2o4gpu
```

Comment:
update this to a hyperlink in markdown: [https://github.com/h2oai/h2o4gpu](https://github.com/h2oai/h2o4gpu)